modules/issue_tracker/test/issue_tracker_test_plan.md

```diff
@@ -6,16 +6,21 @@ Issue Tracker Filter Form [Automation Testing]
 5. Test that the watching checkbox works correctly (issues that your userID is watching in issues_watching table)
 6. Check that links to issues in table are correct.
 7. Check that table sorts and displays additional pages correctly 
+8. Check that a user who does not have access_all_profiles permission and limited sites access (see __user_psc_rel__ table for user/CenterID relationships) can see all issues with a site set to NULL.
```

Comment:
```suggestion
8. Check that a user who does not have access_all_profiles permission and belongs to only one site can see all issues with a site set to NULL.
```
@laemtl editing this because the User Accounts module is used to check and manage User-Site relationships via the front-end. (No need to check/update the `user_psc_rel` table manually in mysql)